diff --git a/node_modules/@polkadot/hw-ledger/cjs/defaults.js b/node_modules/@polkadot/hw-ledger/cjs/defaults.js
index 94cf3de..939134b 100644
--- a/node_modules/@polkadot/hw-ledger/cjs/defaults.js
+++ b/node_modules/@polkadot/hw-ledger/cjs/defaults.js
@@ -6,6 +6,7 @@ exports.ledgerApps = {
     ajuna: 'Ajuna',
     'aleph-node': 'AlephZero',
     astar: 'Astar',
+    'avail-turing': 'Avail',
     bifrost: 'Bifrost',
     'bifrost-kusama': 'BifrostKusama',
     centrifuge: 'Centrifuge',
diff --git a/node_modules/@polkadot/hw-ledger/defaults.js b/node_modules/@polkadot/hw-ledger/defaults.js
index 9f06432..334edfc 100644
--- a/node_modules/@polkadot/hw-ledger/defaults.js
+++ b/node_modules/@polkadot/hw-ledger/defaults.js
@@ -3,6 +3,7 @@ export const ledgerApps = {
     ajuna: 'Ajuna',
     'aleph-node': 'AlephZero',
     astar: 'Astar',
+    'avail-turing': 'Avail',
     bifrost: 'Bifrost',
     'bifrost-kusama': 'BifrostKusama',
     centrifuge: 'Centrifuge',
